### DISCUSSION TITLE
#if UNITY_EDITOR 이런 지정은 가능한가요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#648] [5. Q&A]
#if UNITY_EDITOR 우선 사용해본 바로는 불가능한데, 방법이 있을까요?

**Comment by minjujuu** [comment]
if (Application.isEditor) { } 
저는 위 조건문 사용해서 에디터인지 구분하고 있어요!

**Reply by ryusiin** [reply]
이거 사용했어요, 감사합니다:>
